proto/scripts/swagger.py

```diff
@@ -20,6 +20,10 @@ def merge_dict(d1: Dict, d2: Dict) -> None:
     """
 
     for k, v2 in d2.items():
+        # Nones represent a deletion
```

Comment:
update the docstring for this function too, please.  I was rushing when I wrote this part